addons/mail/static/src/scss/composer.scss

```diff
@@ -384,6 +384,11 @@ $o-mail-attachment-margin: 5px;
         transition: all 0.1s ease 0s;
         transform: translateX(0);
     }
+
+    .o_attachment_delete_cross {
+        @include o-position-absolute($right: 6px, $top:7px);
```

Comment:
There is already a style on `.o_attachment_delete_cross` defined in this file. I would add this 2 style rules there.

Reply:
Yes, there is `.o_attachment_delete_cross` defined in this file but it's inside `.o_image_overlay`

If we add this 2 styles over there, only image attachment will be affected but we want to fix this in all the attachments, so I've added those separately.

Let me know your thoughts.

Reply:
Oh I see, you are right. I didn't check non-image images but it should also work for them too.
Let's keep it that way then!